src/sonic-config-engine/redis_bcc.py

```diff
@@ -0,0 +1,26 @@
+import jinja2
+
+class RedisBytecodeCache(jinja2.BytecodeCache):
+    """ A bytecode cache for jinja2 template that stores bytecode in Redis """
+
+    REDIS_HASH = 'JINJA2_CACHE'
+
+    def __init__(self, client):
+        self._client = client
+        try:
+            self._client.connect(self._client.STATE_DB, retry_on=False)
```

Comment:
>STATE_DB [](start = 46, length = 8)

use another non-used database?

Reply:
Can I use db index 8 and update swss-common/swss-sdk?

Reply:
This is still applicable.

---
In reply to: [338815799](https://github.com/Azure/sonic-buildimage/pull/3658#discussion_r338815799) [](ancestors = 338815799)